extensions/rich_text_components/Math/Math.py

```diff
@@ -32,6 +32,7 @@ class Math(base.BaseRichTextComponent):
         'schema': {
             'type': 'custom',
             'obj_type': 'MathLatexString',
+            'placeholder_text': '\\frac{x}{y}'
```

Comment:
In one case you are binding to a variable and in the other case you are binding to a raw value. As you can see, it works with the former rather than the latter.

So, you could create a constant containing the `\frac{x}{y}` string, and bind to that constant.